README.md

```diff
@@ -27,3 +27,15 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+
+Answer: 
+A good invariant for this would be an array that can hold all the numbers within the sequence. I used the (arr[i-1] + arr[i-2]). The reason I chose this is because it will always be able to hold to elements from the array until I can get the third element. This is always in the same sequence with each iteration until it is stopped or hits the limit. Especially starting with an array [0,1] it should remain true. This will ensure that at the beginning of each iteration, we have the necessary elements to compute the next one. 
+
+To clarify what I am saying above. The index = i and we expect the previous to elements from arr[i-1] and arr[i-2] to be the correct Fibonacci numbers. 
```

Comment:
...so then this should be your invariant?

Reply:
I changed my code to the recursive as asked. I also clarified my invariant because I did not think that in-depth with it.